src/pages/blog/ce-qui-caractérise-le-top-20%-des-devs.md

```diff
@@ -0,0 +1,97 @@
+
+Si vous en doutez regarder autour de vous la grande majorité des logiciels qui marchent réellement dont les gens veulent et qui résoudent des problèmes puis demander vous s’ils auraient eu la même portée coder par un excellent dev certes mais tout seul dans sa grotte ou mal accompagné.
+
+Votre compétence marginale aussi impressionnante soit-elle trouvera ses limites face à une équipe même moyenne mais soudée et qui travail correctement en équipe. Le marché tranchera et cela peut faire mal.
+
+### Focus sur l'impact
+
+Ils comprennent l'importance de l'impact de leur travail sur le produit final et sur l'utilisateur/client. Ils sont motivés par la création de valeur réelle, plutôt que par la simple écriture de code.
+
+C’est un point auquel, je suis particulièrement sensible. Au début, on passe quasi tous par la phase “pisseurs de code” comme on dit dans le jargon. Personnellement, je ne la vois pas systématiquement que d’un mauvais œil, c’est une partie intégrante d’une phase de progression, l’erreur serait d’y rester pire de penser que plus de code impressionnant on envoi, plus on est excellente.
+
+Une phrase de devs seniors avec qui j’ai échangé et qui m’a beaucoup marqué est la suivante :
+
+> “La meilleure ligne de code est celle que l’on n'écrit pas.”
+
+### Pragmatisme
+
+Les développeurs du top 20% sont pragmatiques. Ils savent faire la part des choses entre perfection et productivité. Ils comprennent quand il est nécessaire de faire des compromis pour avancer plus rapidement sans sacrifier la qualité du produit.
+
+C’est certainement la caractéristique sur laquelle j’aurais à le plus fournir d’effort. Etant à l’extrême perfectionniste, je vois littéralement comment cela est handicapant au quotidien. Si vous voulez faire partir des meilleurs vous devrez entrainer ce “soft skill” qui changera votre vie d’entrepreneur, de collaborateur, de freelance, etc. Pensez y vraiment, c’est dur mais nécessaire pour faire partir de la crème.
+
+### Responsabilité
+
+Enfin, ils prennent la responsabilité de leur travail. Ils sont conscients que chaque ligne de code qu'ils écrivent peut avoir des conséquences, et ils s'efforcent de produire un travail de qualité pour minimiser les bugs et les problèmes à long terme.
+
+C’est un sujet qui rejoint la partie caractéristique technique du respect/maitrise des bonnes pratiques du métier mais avec une touche d’éthique qui fait réellement la différence.
+
+Si comme moi, vous êtes naturellement à la recherche de sens dans tout ce que vous faites. Vous vous y trouverez forcément. Et si ce n’est pas le cas, vous devriez sérieusement y songer. Votre vous du futur dira merci à celui du présent, et cela entrainera un cercle vertueux qui fera de vous un dev du top tier. Autant pour vous même (confiance en soi, fierté du travail bien fait, se sentir utile, etc) mais aussi pour le monde extérieur, pairs-utilisateurs-clients (reconnaissance, utilisateurs satisfaits, évitement du pire, etc).
+
+## Conclusion
```

Comment:
You can use another title. Ask yourself these questions?
- What do you want to do here?
- Which ideas do you want to communicate?

Reply:
Done. 
Don't hesitate to ask if you think I should do more. 
The idea is to show that a responsible developer generally has more reasons to get involved in his work and therefore increases his chances of being one of the best. 
I think this is generally a pattern found in people who excel (I could be wrong, there could be exceptions). 
But every developer who evolves will at some point look for meaning in what he does. In my opinion, this is one of the best fuels for long-term success and high-level excellence.